Packages/vcs/vcs/VTKPlots.py

```diff
@@ -1553,3 +1553,9 @@ def png_dimensions(self, path):
         img = reader.GetOutput()
         size = img.GetDimensions()
         return size[0], size[1]
+
+    def canvasraised(self):
```

Comment:
It is the VTK implementation of [this](https://github.com/UV-CDAT/uvcdat/blob/350516e7fe7fa36d0cf3452a8cc3bc3104319a1b/Packages/vcs/vcs/Canvas.py#L5268-L5272) function, which previously said this:

    ```
#############################################################################
    #                                                                           #
    # Raise VCS Canvas to the top of all its siblings.                          #
    #                                                                           #
    #############################################################################
    def canvasraised(self, *args):
        """
 Function: canvasraised                         # Raise the VCS Canvas to the top
 Description of Function:
    This function marks a VCS Canvas as eligible to be displayed and  
    positions the window at the top of the stack of its siblings.
 Example of Use:
    a=vcs.init()
    ...
    a.canvasraised()
"""

        return  apply(self.canvas.canvasraised, args)
```

https://github.com/UV-CDAT/uvcdat/blob/303c02a32095aa328b5aeba53ef20267e49a63c4/Packages/vcs/Lib/Canvas.py#L7819-L7839